src/sentry/middleware/user.py

```diff
@@ -1,17 +1,19 @@
 from __future__ import absolute_import
 
-from django.utils import timezone
 from datetime import timedelta
-import inspect
 
+from django.utils import timezone
+from sentry.api.authentication import TokenAuthentication
 
-class UserActiveMiddleware(object):
-    allowed_paths = (
-        'sentry.api.endpoints',
-        'sentry.web.frontend',
-    )
 
+class UserActiveMiddleware(object):
     def process_view(self, request, view_func, view_args, view_kwargs):
+        try:
+            auth = TokenAuthentication()
+            request.user = auth.authenticate(request)[0]
```

Comment:
Hey dcramer- take a look at the screencast of a replay with the auth middleware. It looks like get_user() in auth.py is handling the session auth. Thoughts?

[active_user_authMW.zip](https://github.com/getsentry/sentry/files/1106741/active_user_authMW.zip)